user_docs/en/userGuide.md

```diff
@@ -3387,6 +3387,14 @@ It is very important to only install add-ons from sources you trust.
 The functionality of add-ons is unrestricted inside NVDA.
 This could include accessing your personal data or even the entire system.
 
+As with any software, it is important to trust the developer of an add-on before using it.

```

Comment:
This line feels like a duplicate of 3386 "It is very important to only install add-ons from sources you trust."

Reply:
```suggestion
```